app.js

```diff
@@ -100,3 +105,6 @@ oppiaGithubPages.controller('theExplorationGalleryController', function($scope)
 oppiaGithubPages.controller('learnerViewController', function($scope) {
 
 });
+oppiaGithubPages.controller('planningYourExplorationController', function($scope) {
```

Comment:
OK, I'd say let's get rid of them, since they currently add complexity and are not needed. As you say, we can add them later.